README.md

```diff
@@ -2,87 +2,100 @@
+  <a href="https://github.com/CodeforAustralia/council-emissions-calculator/commits/staging"><img alt="Commit history" src="https://img.shields.io/github/commit-activity/m/CodeforAustralia/council-emissions-calculator.svg"></a>
+  <a href="https://github.com/CodeforAustralia/council-emissions-calculator/graphs/contributors"><img alt="Contributors" src="https://img.shields.io/badge/all_contributors-9-blue.svg?style=flat-square"></a>
 </p>
 
-<p align="center">
-  <a href="#Project-Goals">Project Goals</a> •
-  <a href="#Getting-Started">Getting Started</a> •
-  <a href="#Learn-More-about-Development"> Development</a> •
-  <a href="#Collaboration-Working-In-a-Forked-Repository">Collaboration </a>
-</p>
+
+## Overview
+
+|Category|Details|
+|--------|-------|
+| Purpose | Provide a source of information that facilitates the decision process for councils to become more sustainable |
+| Custodian | Civic Makers |
+| Slack | \#civic-makers-climatechange |
+| Tech Stack | Next JS, Chakra UI, Google Sheets |
+| Design | https://www.figma.com/file/Y42AKuHDW46LiLDaTOs6sM/Heroes-carbon-calculator-designs?node-id=978%3A54391&t=jSZfGy4RjwTVhtH4-1 |
+| Staging Endpoint | http://council-emissions-calculator-spike-flax.vercel.app/ |
+| Prod Endpoint | https://seechange.codeforaustralia.org/ |
+
+## Table of Contents
+
+- [Project Goals](#project-goals)
+- [System Diagram](#system-diagram)
+- [Getting Started](#getting-started)
+- [Development](#getting-started)
+- [Resourses](#resourses)
+- [Dependencies](#dependencies)
 
 ## Project Goals
 
-- Provide councils a tool to assess their impact on the environment and ...
-- Provide councils with inspirations and next steps to have a more sustainable way of working.
-- Provide councils with inspirations and next steps to engage communities in initiatives that are favorable for the environment.
+- Provide councils a tool to assess their impact on the environment.
+- Provide councils with inspirations and next steps to have a more sustainable way of workingand engage communities in initiatives that are favorable for the environment.
 - Provide a source of information that facilitates the decision process for councils to become more sustainable.
-- to learn more about this project, check out our [project planning](https://docs.google.com/document/d/1h5r1AfbQC8Azy_cdGHGBGOJwSxSqvt0cHKd5MmSd7qo/edit#heading=h.fw4lq2wclkt9) and [trello project board.](https://trello.com/b/ZXaIkclp/civic-makers-2021-l-use-this-one)
+
+To learn more about this project, check out our [project planning](https://docs.google.com/document/d/1h5r1AfbQC8Azy_cdGHGBGOJwSxSqvt0cHKd5MmSd7qo/edit#heading=h.fw4lq2wclkt9) and [trello project board.](https://trello.com/b/iPHuVc0J/civic-makers-2023)
+
+## System Diagram
+
+![Council carbon emission calculator system diagram](https://user-images.githubusercontent.com/88268603/232468816-142513de-c835-4dd3-923f-416635711103.png)
```

Comment:
I **love** your systems diagram @OlaShabalina !!! 😍 

A couple of suggestions for improvement I offer for your thoughts:
* the survey results are stored and processed in *Google Sheets*, not *Google Pages* 😅 
* Did you create your diagram with DrawIO (or Excalidraw)? Are you able to put a copy of the DrawIO file *and* the PNG image file in the repo? I would love to have your diagrams version controlled as well! 🙇🏽

Reply:
Loaded the file and updated Google Sheets